SECURITY.md

```diff
@@ -0,0 +1,59 @@
+<!--
+Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+-->
+<!-- BEGIN PRAVEGA SECURITY.MD V0.0.1 BLOCK -->
+
+## Security
+
+Pravega takes the security of our software products and services seriously, which includes all source code repositories managed through our GitHub organizations.
+
+If you believe you have found a security vulnerability in any Pravega owned repository, please report it to us as described below.
+
+## Reporting Security Issues
+
+**Please do not report security vulnerabilities through public GitHub issues.**
+
+Instead, please report them to the Pravega Security Team via email(mailto:security@pravega.io).
+
+You should receive a response within 24 hours. If for some reason you do not, please follow up via email to ensure we received your original message.
```

Comment:
I am not sure about "receive a response within 24 hours". I don't think we should promise that. These two lines could be removed.

Reply:
CII Best Practices stipulates 14 days to respond: https://github.com/pravega/pravega/pull/5052#discussion_r472357416